### ISSUE TITLE
Age 3 - Demon Crystallizer uncraftable

### ISSUE THREAD

**Issue by shane3silbaugh** [DarkPacks/SevTech-Ages#4337]
## Bug Description

Playing SevTech 3.1.7, Age 3 unlocked, the Blood Magic Demon Crystallizer is listed as craft-able. But the only recipe listed requires a Manyullyn ingot which is locked in Age 4.

### Did this behavior use to work in the previous version?

I re-loaded 3.0.8 SevTech and confirmed that in that version there are 2 recipes for the Demon Crystallizer. One using a Manyullyn ingot and one using Lapis Lazuli

## Steps to Reproduce (for bugs)

1. Load up 3.0.8, grant all achievements up to Age 3, see 2 recipes for Demon Crystallizer
2. Load up 3.1.7, grant all achievements up to Age 3, see 1 recipe for Demon Crystallizer
3.

## Logs

* Client/Server Log: Seems like a config issue to me, if you need logs ask.
* Crash Log: Ditto

## Client Information
* Modpack Version: 3.1.7
* Java Version: 1.8.0_261
* Launcher Used: Twitch
* Memory Allocated: 7,936 MB
* Server/LAN/Single Player: Server and Single Player
* Optifine Installed:
* Shaders Enabled:

## World Information
* Modpack Version world created in: 3.1.7
* Additional Content Installed: None

**Comment by GHuser357** [comment]
Manyullyn is obtainable in Age 3 via Blood Magic. Ardite and Cobalt can be obtained from the Nether with the Ritual of Magnetism/Crushing. Crush them in the crusher, then alloy in the arcfurnace. Unfamiliar items can still be used for crafting, you just can't put them in your hand. There are many unfamiliar items in the game that can be obtained and still used for crafting. You can obtain all the space iron ores in Age 3 with the Abyssalcraft Materializer.

Edit: looks like the recipe is now a platinum ingot.

**Comment by shane3silbaugh** [comment]
In 3.1.7 Ardite, ~~Cobolt~~ Cobalt, and Manyullyn are all listed as Unknown Stage: Four.

If it's been changed to Platinum in a future patch that's good. I'll look for that.

**Comment by GHuser357** [comment]
> In 3.1.7 Ardite, Cobolt, and Manyullyn are all listed as Unknown Stage: Four.

Yes I know they are but they are still obtainable in age 3 via the method I posted.

> If it's been changed to Platinum in a future patch that's good. I'll look for that.

Edit: Sorry my mistake the Demon Crucible has a platinum ingot

You can still make the Demon Crystalizer in Age 3 with a manyullyn ingot.

I just made one in Age 3 version 3.1.7. It works.

You can also obtain a diamond block for under your ritual of magnetism.

**Comment by shane3silbaugh** [comment]
Thank you, but using Ritual of Magnetism to mine up an ore the developers specifically limited to stage 4 in the configs seems a bit like cheating to me. If it doesn't feel that way to you, that's fine. I just wanted this third play through to follow the intent of the developers and not use anything from future Stages even though I might be able to get around them. 

The configs that I checked listed the Demon Crystallizer as specifically placed in Stage 3 and the Manyullyn Ingots as specifically in Stage 4. Since there was a previous recipe with all Stage 3 items available in an older version of SevTech this seems to be a bug that the developers will want to fix.


--
BTW, I do use the Ritual of Magnetism and the Ritual of Crushing to mine. But I had forgotten that I could use that in the Nether to mine up Staged Ore. And If I wanted just get around the staging, I could use the Neromantic Prime and Evershifting Fountain to find Overworld chunks with Molten Cobalt or Molten Ardite. Though that would make it much harder because the Evershifting Fountain is a pain to setup...

**Comment by shane3silbaugh** [comment]
$#@, hit the wrong button updating my ticket

**Comment by GHuser357** [comment]
> seems a bit like cheating to me.

It's Minecraft, the only rules are the ones you make for yourself. It's supposed to be fun. Anyways, good luck and have fun.
